isubit/iastate_theme
# D8ISUTHEME-143 Add Discord icon

This pull request adds a custom icon font that provides a Discord icon. It works the same way as we're hosting our own copy of academicons. 

To test:
1. Add a link to a discord server/invite (or just use discord.com) to the social media menus. Those are the one in the footer and the one in the People Profile.
2. Add some other social icons along with them.
3. Confirm the icon appears and it's sized and aligned correctly.
